backend/src/middleware/errorHandler.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from 'express';
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/47#issuecomment-2436111848>#96fecf</a><span> - 10/24/2024, 07:07 pm</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Consider adding a `next` parameter to the `errorHandler` function to conform to Express error handling middleware signature. This might be necessary for proper error handling in the Express middleware chain.
</div>
<div id="fix">
<b>Fix:</b> Add the 'next' parameter to the 'errorHandler' function signature and import the NextFunction type from express.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,5 +1,5 @@
 -import { Request, Response } from 'express';
 +import { Request, Response, NextFunction } from 'express';
  import logger from '../utils/logger';
  import { AppError } from '../utils/errors';
 
 -const errorHandler = (err: AppError | Error, req: Request, res: Response) => {
 +const errorHandler = (err: AppError | Error, req: Request, res: Response, next: NextFunction) => {
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged